### PR TITLE
Removed no-underscore-dangle rule

### PR DESCRIPTION
**Pull request by srounce**
Is there a good reason why this is explicitly set?

### CODE REVIEW

File: index.js
Comment (colinclarkbcgdv) [inline index.js]: because it overrides airbnb's rule that allows trailing commas

> https://github.com/airbnb/javascript/blob/8cab89e792e2fc457ab177cdf2d97e767e03ecb6/README.md#commas--dangling

**Review by colinclarkbcgdv — CHANGES_REQUESTED** [review]

File: index.js
Comment (colinclarkbcgdv) [reply index.js]: so we should probably keep that for now

Comment (srounce) [reply index.js]: Derp wrong rule.

**Review by colinclarkbcgdv — APPROVED** [review]
_cool, er, cool

**Comment by srounce** [comment]
@zimbatm merge/permissions plz

**Review by colinclarkbcgdv — DISMISSED** [review]
Wait, this allows underscores, so am removing the approval. https://github.com/airbnb/javascript/blob/8cab89e792e2fc457ab177cdf2d97e767e03ecb6/README.md#naming--leading-underscore

I think you should wait to get consensus on this PR before merging it.

**Comment by zimbatm** [comment]
The Airbnb style guide kind of shoves the issue under the rug. The only other approach I know of is to hide private variables in a function scope. I'm not convinced this is a preferable solution though:

```js
function newCounter() {
  var counter = 0;
  return {
    incr: function() { counter += 1; },
    read: function() { return counter; } 
  };
}
```

**Review by colinclarkbcgdv — APPROVED** [review]
should not be blocked

**Comment by srounce** [comment]
@eadz @jaiohanlon @jamieallen59 @rcambrj @martingBCG 
Do you have any input?

**Comment by srounce** [comment]
@zimbatm I don't have permission to merge 😄

**Comment by zimbatm** [comment]
Permission from whom?

**Comment by srounce** [comment]
"Need write access… blah"

**Comment by zimbatm** [comment]
Actually another option would be to introduce the `$` as a private prefix like you did in the wonderbill-api code.

**Comment by rcambrj** [comment]
underscores have been used for private stuff in the extension since day one 

I know I'm late to the party, but :+1:

**Pull request merged**
